hyperspy/_components/rc.py

```diff
@@ -16,24 +16,52 @@
 # You should have received a copy of the GNU General Public License
 # along with  HyperSpy.  If not, see <http://www.gnu.org/licenses/>.
 
-import numpy as np
+from hyperspy._components.expression import Expression
 
-from hyperspy.component import Component
+class RC(Expression):
 
+    r"""
+    RC function component (based on the time-domain capacitor voltage response 
+    of an RC-circuit)
+    
+    .. math::
 
-class RC(Component):
-
-    """
+        f(x) = V_\mathrm{0} + V_\mathrm{max} \left[1 - \mathrm{exp}\left(
+            -\frac{x}{\tau}\right)\right]
+    
+    ====================== ===========
+    Variable                Parameter 
+    ====================== ===========
+    :math:`V_\mathrm{0}`    V0     
+    :math:`V_\mathrm{max}`  Vmax    
+    :math:`\tau`            tau    
+    ====================== ===========
+    
+    
+    Parameters
+    -----------
+    V0: float
+        vertical offset
+    Vmax: float
+        maximum voltage, asymptote of the function for 
+        :math:`\mathrm{lim}_{x\to\infty}`
+    tau: float
+        tau=RC is the RC circuit time constant (voltage rise time)
+    **kwargs
+        Extra keyword arguments are passed to the ``Expression`` component.
+    
     """
 
-    def __init__(self, V=1, V0=0, tau=1.):
-        Component.__init__(self, ('Vmax', 'V0', 'tau'))
-        self.Vmax.value, self.V0.value, self.tau.value = V, V0, tau
-
-    def function(self, x):
-        """
-        """
-        Vmax = self.Vmax.value
-        V0 = self.V0.value
-        tau = self.tau.value
-        return V0 + Vmax * (1 - np.exp(-x / tau))
+    def __init__(self, Vmax=1., V0=0., tau=1., module="numexpr", **kwargs):
+        super(RC, self).__init__(
+            expression="V0 + Vmax * (1 - exp(-x / tau))",
+            name="RC",
+            Vmax=Vmax,
+            V0=V0,
+            tau=tau,
+            module=module,
+            autodoc=False,
+            **kwargs,
+        )
+        
+        self.isbackground = True
```

Comment:
Out of curiosity, what is the reason to consider this component as a background?

Reply:
Good point, must have sneaked in from a copy-paste. Does not make much sense for an RC function (whereas it would indeed make sense for the skew-normal).

Reply:
This is only used for EDS and EELS model, but I guess this is better to remove it completely.

Reply:
Now set to False.